### PR TITLE
Update for log API changes

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
$subject

### CODE REVIEW

**Review by daneshk — APPROVED** [review]

File: auth-ballerina/utils.bal [hunk 1]
```diff
@@ -140,7 +140,7 @@ isolated function cacheAuthzResult(boolean authorized, string authzCacheKey, cac
         if (pCache is cache:Cache) {
             cache:Error? result = pCache.put(authzCacheKey, authorized);
             if (result is cache:Error) {
-                log:printDebug("Failed to add entry to positive authz cache");
+                log:print("Failed to add entry to positive authz cache");
```
Comment (shafreenAnfar) [inline auth-ballerina/utils.bal:143]: When do we get this error ?

Comment (ldclakmal) [reply auth-ballerina/utils.bal:143]: When we try to add some entry to authorization cache and failed due to an error of the ballerina/cache.

Comment (ldclakmal) [reply auth-ballerina/utils.bal:143]: This shouldn't happen actually. If that happened due to some reason, the authorization cache doesn't work as expected. In order to find that from auth module perspective, thought of keeping this log. WDYT?

Comment (shafreenAnfar) [reply auth-ballerina/utils.bal:143]: Yeah, but why does it throw an error ?

Comment (shafreenAnfar) [reply auth-ballerina/utils.bal:143]: IMO, in that case it should be an error

Comment (ldclakmal) [reply auth-ballerina/utils.bal:143]: I will update to the error log.

**Pull request merged**